appveyor.yml

```diff
@@ -5,6 +5,11 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe -> appveyor.yaml
+  - NsProcess.zip -> appveyor.yaml
+  - thirdparty\windows -> thirdparty\fetch-thirdparty-deps.ps1

```

Comment:
They are the largest ones and I am not sure what happens when you cache a directory that also contains the scripts and some static DLLs. https://github.com/OpenRA/OpenRA/pull/7953 was not on the horizon back then when I filed this.